test/runtests.jl

```diff
@@ -6,6 +6,8 @@
 # run unit-tests
 #############################################################################
 
+push!(LOAD_PATH, "src")
```

Comment:
Is all the code under the `~/.julia/v0.4/StochDynamicProgramming/` folder? You shouldn't need this line. `using StochDynamicProgramming` should be sufficient. Then you can either reference unexported methods with the `StochDynamicProgramming.` prefix or export them.